src/features/Send/common/SendContext.tsx

```diff
@@ -274,11 +275,16 @@ const getTotalUsedByOtherTargets = ({
   }, Quantities.zero)
 }
 
-export const useSelectedTokensCounter = () => {
+export const useSelectedSecondaryAmountsCounter = (wallet: YoroiWallet) => {
   const {targets} = useSend()
-  const selectedTokensCounter = targets.reduce((acc, target) => {
-    return Amounts.toArray(target.entry.amounts).length + acc
-  }, 0)
+  const isSecondaryAmount = isSecondaryAmountFilter(wallet)
 
-  return selectedTokensCounter
+  return targets.reduce((acc, target) => {
+    return Amounts.toArray(target.entry.amounts).filter(isSecondaryAmount).length + acc
+  }, 0)
 }
+
+const isSecondaryAmountFilter =
```

Comment:
`isSecondaryAssetFilter` in this case?

Reply:
there is no asset for yoroi, we have TokenInfo, Quantity(ies) and Amount(s) which is {[tokenId]: Quantity}, tx contains Amounts, therefore you add Amount(s) -> Txs.